app/src/main/java/com/cdio/solitaire/controller/StrategyController.kt

```diff
@@ -210,80 +212,259 @@ object StrategyController {
+//                                        } else {
+//                                            tempgsc.performMove(tempMove)
+//                                        }
+//                                        tempMove = tempMove.next
+//                                    }
+//                                    //Now tempGameState should be updated with the desired move.
+//                                    //Need to check if there is a possible move from talon.
+//                                    val listOfCardsInReachInTempGameState =
+//                                        getListOfCardsInReach(tempgsc)
+//                                    for (talonCard in listOfCardsInReachInTempGameState) {
+//                                        cardNotFound = true
+//                                        while (cardNotFound) {
+//                                            if (tempgsc.gameState.stock.size >= 3) {
+//                                                val move = Move(MoveType.DRAW_STOCK)
+//                                                tempgsc.performMove(move)
+//                                                if (tempgsc.gameState.talon.tail != null && tempgsc.gameState.talon.tail!!.rank == talonCard.rank && tempgsc.gameState.talon.tail!!.suit == talonCard.suit) {
+//                                                    cardNotFound = false
+//                                                    for (column in tempgsc.gameState.tableaux) {
+//                                                        val move = Move(
+//                                                            MoveType.MOVE_FROM_TALON,
+//                                                            targetStack = column,
+//                                                            sourceCard = tempgsc.gameState.talon.tail,
+//                                                            sourceStack = tempgsc.gameState.talon
+//                                                        )
+//                                                        if (tempgsc.isMoveLegal(move)) {
+//                                                            return moveQueue
+//                                                        }
+//                                                    }
+//                                                }
+//                                            } else {
+//                                                tempgsc.performMove(Move(MoveType.FLIP_TALON))
+//                                                val move = Move(MoveType.FLIP_TALON)
+//                                            }
+//                                        }
+//                                    }
+//                                }
+//                            }
+//                        }
+//                    }
+//                }
+//            }
+//            return null
+//        } else {
```

Comment:
Remove commented out code? Unless you need it later.

Reply:
Not fully implemented. Plan is to work on it later.